### ISSUE TITLE
Replace dummy files in tools with the actual binaries

### ISSUE THREAD

**Issue by lhallam** [d0k3/Universal-Inject-Generator#16]
Binaries need to be added for `tools/3dstool-Linux-x86_64`, `tools/MergeExHeader-Linux-x86_64`, and `tools/ctrtool-Linux-i686`.

Currently, running process.sh on linux x86_64 will fail without showing the errors in the dummy files:

```
--- UNIVERSAL INJECT GENERATOR v0.6 ---
--- LINUX EDITION ---


[+] IDENTIFY SYSTEM ARCHITECTURE
[+] IDENTIFY FILES TO WORK WITH
[+] EXTRACT HS AND INJECT APP
[+] GENERATE NO BANNER EXEFS
cp: cannot stat ‘work/hs_exefs/banner.bnr’: No such file or directory
[+] GENERATE DUMMY ROMFS
mv: cannot stat ‘work/dummy_romfs.bin’: No such file or directory
[+] MERGE EXHEADER
[+] REBUILD HS INJECT APP
ls: cannot access input/FBI_inject_no_banner.app: No such file or directory
ls: cannot access input/FBI_inject_with_banner.app: No such file or directory
mv: cannot stat ‘input/FBI_inject_no_banner.app’: No such file or directory
mv: cannot stat ‘input/FBI_inject_with_banner.app’: No such file or directory
[+] HS APP ORIGINAL SIZE : 907264 bytes
[+] HS APP INJECT (N) SIZE: bytes
tools/process.sh: line 68: [: 907264: unary operator expected
[+] HS APP INJECT (B) SIZE: bytes
tools/process.sh: line 74: [: 907264: unary operator expected
```

A thread on gbatemp where someone has this issue: https://gbatemp.net/threads/linux-universal-inject-generator-doesnt-work.428315/

**Comment by mariogamer2** [comment]
https://github.com/d0k3/Universal-Inject-Generator/pull/17 partly fix this, the only thing who must be generated is 3dstool for x86 arch...

**Comment by lhallam** [comment]
Fixed by #17
